src/Compilers/CSharp/Portable/BoundTree/UnboundLambda.cs

```diff
@@ -191,7 +193,7 @@ public TypeWithAnnotations GetInferredReturnType(ConversionsBase conversions, Nu
                     // Need to handle ref returns. See https://github.com/dotnet/roslyn/issues/30432
                     if (conversions.IncludeNullability)
                     {
-                        bestResultType = NullableWalker.BestTypeForLambdaReturns(returns, compilation, node);
+                        bestResultType = NullableWalker.BestTypeForLambdaReturns(returns, compilation, node, (Conversions)conversions);
```

Comment:
>(Conversions) [](start = 109, length = 13)

Can the method argument be changed to `Conversions`?

Reply:
Added a prototype comment to run on all tests and throw the results away, as I think there might be a case where this will throw.

---
In reply to: [269163031](https://github.com/dotnet/roslyn/pull/34429#discussion_r269163031) [](ancestors = 269163031)